.github/workflows/scripts/start-kube-burner.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
+set -euox pipefail
+
+export KUBE_BURNER_VERSION=1.4.3
+
+mkdir -p ./kube-burner
+
+curl --silent --location "https://github.com/cloud-bulldozer/kube-burner/releases/download/v${KUBE_BURNER_VERSION}/kube-burner-${KUBE_BURNER_VERSION}-$(uname -s)-$(uname -m).tar.gz" --output "./kube-burner/kube-burner-${KUBE_BURNER_VERSION}.tar.gz"
+
+tar -zxvf "./kube-burner/kube-burner-${KUBE_BURNER_VERSION}.tar.gz" --directory ./kube-burner
+
+kube_burner_config_file="$STACKROX_DIR"/.github/workflows/kube-burner-configs/cluster-density-kube-burner.yml 
+kube_burner_gen_config_file="$STACKROX_DIR"/.github/workflows/kube-burner-configs/cluster-density-kube-burner_gen.yml 
+
+sed "s|STACKROX_DIR|$STACKROX_DIR|" "$kube_burner_config_file" > "$kube_burner_gen_config_file" 
+
+nohup "$STACKROX_DIR"/.github/workflows/scripts/repeate-kube-burner.sh ./kube-burner/kube-burner "$kube_burner_gen_config_file" &
```

Comment:
The problem here is that this exits immediately and then kube-burner doesn't fully run. This script works locally but not in github actions. If this is not run in the background than the github actions doesn't end. Kube-burner could be run in a VM. The script to do that would be bit a more complicated. What is the best option.

Reply:
Could you run kube-burner as a K8s Job on the GKE cluster?
The endless loop wouldn't need to run in the background, instead as the process of the job container, which will stay running as long as the cluster exists.

Reply:
I am now running kube-burner as a K8s job. While it creates namespaces, it doesn't create deployments. I am working on fixing that.